corehq/toggles.py

```diff
@@ -557,13 +587,20 @@ def toggle_values_by_name(username=None, domain=None):
     ("Blacklist API access to a user or domain that spams us"),
     TAG_EXPERIMENTAL,
     namespaces=[NAMESPACE_DOMAIN, NAMESPACE_USER],
+    description="For temporary, emergency use only. If a partner doesn't properly "
+    "throttle their API requests, it can hammer our infrastructure, causing "
+    "outages. This will cut off the tide, but we should communicate with them "
+    "immediately.",
 )
 
 FORM_SUBMISSION_BLACKLIST = StaticToggle(
     'FORM_SUBMISSION_BLACKLIST',
     ("Blacklist form submissions from a domain that spams us"),
     TAG_EXPERIMENTAL,
     namespaces=[NAMESPACE_DOMAIN],
+    description="This is a temporary solution to an unusually high volume of "
+    "form submissions from a domain.  We have some projects that automatically "
+    "send forms. If that ever causes problems, we can use this to cut them off.",
```

Comment:
We've actually had the ability to add a verbose description of a feature flag for a while.  Perhaps someone from @dimagi/product would be interested in taking a pass at existing feature flags and adding descriptions?